src/main/java/hw4/components/LeftSide.java

```diff
@@ -0,0 +1,42 @@
+package hw4.components;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.page;
+
+public class LeftSide {
+
+    @FindBy(id = "mCSB_1")
+    private SelenideElement leftSide;
+
+    @FindBy(xpath = "//ul[@class='sidebar-menu']/li/a/span")
+    private List<SelenideElement> leftPanelMenuItems;
```

Comment:
Why do you use `List<SelenideElement>` instead of `ElementsCollection`

Reply:
fixed